Hypermedia.AspNetCore.Siren/Resources/AddClassesStep.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Hypermedia.AspNetCore.Siren.Resources
+{
+    using Environments;
+    using Hypermedia.AspNetCore.Siren.Entities;
+    using System.Threading.Tasks;
+
+    internal class AddClassesStep : IAsyncBuildStep<IEntityBuilder, IEntity>
+    {
+        private string[] _classes;
+
+        public void Configure(string[] classes)
+        {
+            this._classes = classes;
+        }
+
+        public Task BuildAsync(IEntityBuilder builder)
+        {
+            builder.WithClasses(this._classes);
```

Comment:
We should do a null-check here.

Reply:
Not really necessary, the Configure is allays called. Maybe after Unit Testing this .

Reply:
I was talking about the builder, not "_classes".